setup.py

```diff
@@ -20,12 +20,9 @@
     "pyyaml>=5.1",
     "pandas",
     "glymur",
-    "scikit-learn>=0.23.2",
-    "scikit-image>=0.17",
-    "torchvision==0.9.1",
-    "torch==1.8.1",
+    "torch==1.8.2+cu102",
+    "torchvision==0.9.2+cu102",
     "tqdm==4.60.0",
-    "requests",
```

Comment:
why requests is removed?

Reply:
Honestly, I don't know! I didn't touch the setup.py file. However, I will include them manually in the next commit.